core/connection.js

```diff
@@ -305,18 +298,18 @@ Blockly.Connection.prototype.connect = function(otherConnection) {
       otherConnection.connect_(this);
     }
     if (!eventGroup) {
-      Blockly.Events.setGroup(false);
+      Events.setGroup(false);
     }
   }
 };
 
 /**
  * Update two connections to target each other.
- * @param {Blockly.Connection} first The first connection to update.
- * @param {Blockly.Connection} second The second connection to update.
+ * @param {Connection} first The first connection to update.
+ * @param {Connection} second The second connection to update.
  * @private
  */
-Blockly.Connection.connectReciprocally_ = function(first, second) {
+Connection.connectReciprocally_ = function(first, second) {
```

Comment:
Since this one is private and static, I think it can just become a module-internal function named `connectReciprocally`.

Reply:
Same for the next few private static functions.

Reply:
Done!